File: How to write an SFTP file transfer client for WinForms/[C#]-How to write an SFTP file transfer client for WinForms/C#/PasswordPrompt.cs
using System;
using System.Windows.Forms;

namespace ClientSample
{
    public partial class PasswordPrompt : Form
    {
        public PasswordPrompt(string title)
        {
            InitializeComponent();

            this.Text = title;
        }

        /// <summary>
        /// Gets the password.
        /// </summary>
        public string Password
        {
            get
            {
                return txtPassword.Text;
            }
        }

        /// <summary>
        /// Handles the OK button's Click event.
        /// </summary>
        /// <param name="sender">The button object.</param>
        /// <param name="e">The event arguments.</param>
        private void btnOk_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
        }
    }
}

File: FTP Client for WinForms using Ultimate FTP library/[VB]-FTP Client for WinForms using Ultimate FTP library/VB/PasswordPrompt.vb
Namespace ClientSample
	Partial Public Class PasswordPrompt
		Inherits Form
		Public Sub New(ByVal title As String)
			InitializeComponent()

			Me.Text = title
		End Sub

		''' <summary>
		''' Gets the password.
		''' </summary>
		Public ReadOnly Property Password() As String
			Get
				Return txtPassword.Text
			End Get
		End Property

		''' <summary>
		''' Handles the OK button's Click event.
		''' </summary>
		''' <param name="sender">The button object.</param>
		''' <param name="e">The event arguments.</param>
		Private Sub btnOk_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnOk.Click
			Me.DialogResult = System.Windows.Forms.DialogResult.OK
		End Sub
	End Class
End Namespace